build.gradle.kts

```
@@ -246,10 +250,21 @@ subprojects {
 
   tasks.withType<Test> {
     useJUnitPlatform()
-    testLogging {
-      events("started", "passed", "skipped", "failed")
-      exceptionFormat = TestExceptionFormat.FULL
-      showStandardStreams = false
+
+    val enableLogging = project.findProperty("misk.test.logging")?.toString()?.toBoolean() ?: true
```

```suggestion
    val enableLogging = project.findProperty("misk.test.logging")?.toString().toBoolean()
```

If we're already setting this to false for the project, we can probably not set true as the null default. `toBoolean` can also be called on null and will return false and be simpler to maintain.